R/birdlife.R

```diff
@@ -25,3 +25,46 @@ birdlife_habitat = function(id){
   out = cbind(id, tables[[habitat_table_number]])
   out[-nrow(out), ] # Drop last row (altitude)
 }
+
+
```

Comment:
Sorry, my intent was to put up the code and then add documentation once it got the thumbs up.  I'll make sure not to push prematurely next time.